sdk/eventhub/event-hubs/src/connectionContext.ts

```diff
@@ -117,6 +124,20 @@ export namespace ConnectionContext {
       audience: options.managementSessionAudience
     };
     connectionContext.managementSession = new ManagementClient(connectionContext, mOptions);
+    connectionContext.isConnectionClosing = function() {
+      // When the connection is not open, but the remote end is open,
+      // then the rhea connection is in the process of terminating.
+      return Boolean(!this.connection.isOpen() && this.connection.isRemoteOpen());
```

Comment:
Is there any case when `isOpen()` would return true and `isRemoteOpen()` would return false?

Reply:
I'd expect this would be possible while the connection is opening and waiting for the service to respond, but I'll confirm.
What is your concern though?

Reply:
So far I've confirmed 1 case where `isOpen() == true && isRemoteOpen() == false`.
1. When opening a connection, before the service has responded that the connection is open.

I'm currently waiting to see what happens if I leave the connection open for a long time (expecting the service to initialize the close).

I also did verify that when the client calls close, `isOpen() == false && isRemoteOpen() == true` until the service reports that it has closed on its end.